### DISCUSSION TITLE
Nice Software but ......

### DISCUSSION THREAD

**Discussion by IK8HCG** [AlexandreRouma/SDRPlusPlus#1054] [General]
I've been trying out this new software ver. 1.1.0 combined with my SDRPLAY1A, it works great compared to SDRUNO, SDR Console and others.
I wanted to ask that the implementation of the zoom, Max and Min be implemented in the way it is set, because if the program is terminated and restarted, zoom, max and Min reset to the Default position. I hope in the implementation of DX cluster

**Comment by AlexandreRouma** [comment]
The min and max values are saved. If they are not you likely installed the software somewhere it doesn't have write access and isn't able to save its config files. The zoom is not saved on purpose as it is reset when the bandwidth of the SDR is changed anyway.

First-party DXcluster implementation is not currently planned.

**Reply by IK8HCG** [reply]
I correct:
Zoom is positioned in default while min and max remain unchanged. the Com and virtual ports configuration would be ideal for interfacing it to software and cw skimmers

**Reply by IK8HCG** [reply]
please Fix in 6 meter (in italian Radiamatori 6 m) now report Radiamatori 9 m
